.github/workflows/InstallRunAndArtifact.yml

```diff
@@ -0,0 +1,285 @@
+        run: |
+          $dataDir = "$(Get-Location)\MetaMorpheus\CMD\bin\Release\net8.0\Data"
+          dotnet "$(Get-Location)\MetaMorpheus\CMD\bin\Release\net8.0\CMD.dll" -s "$dataDir\SmallCalibratible_Yeast.mzML" -d "$dataDir\SmallYeast.fasta" -t "$dataDir\SearchTask.toml" -o "$dataDir\built_dll_output" -v minimal
+          if ($LASTEXITCODE -ne 0) {
+            Write-Host "❌ Built CMD.dll search failed with exit code $LASTEXITCODE" -ForegroundColor Red
+            exit $LASTEXITCODE
+          }
+
+  ## Installs and builds. Verifies that the same dlls exist
+  validate_installer_contents:
+    name: Validate Installer Contents
+    runs-on: windows-latest
+    needs: install
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: 8.0.204
+
+      - name: Download installer artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: MetaMorpheusInstaller
+
+      - name: Install MetaMorpheus MSI
+        run: |
+          Start-Process -FilePath msiexec.exe -ArgumentList "/i `"$(Get-Location)\MetaMorpheusInstaller.msi`" /qn /l*v install.log" -Wait
+
+      - name: Restore dependencies
+        run: dotnet restore ./MetaMorpheus/MetaMorpheus.sln
+
+      - name: Build solution
+        run: dotnet msbuild -v:minimal -p:Configuration=Release -p:UseSharedCompilation=false ./MetaMorpheus/
+
+      - name: Compare built DLLs to installed DLLs
+        run: |
+          $buildDir = $repoDir = Get-Location
+          $installDir = "C:\Program Files\MetaMorpheus"
+
+          # Get DLL names
+          $buildDlls = Get-ChildItem -Path $buildDir -Filter *.dll | Select-Object -ExpandProperty Name
+          $installedDlls = Get-ChildItem -Path $installDir -Filter *.dll | Select-Object -ExpandProperty Name
+
+          # Define whitelist for missing DLLs (build-only) - Okay to not be installed
+          $whitelistedMissingDlls = @(
+            "System.Configuration.ConfigurationManager.dll",
+            "System.Security.Cryptography.ProtectedData.dll",
+            "System.Security.Permissions.dll",
+            "System.Windows.Extensions.dll",
+            "Microsoft.Win32.SystemEvents.dll",
+            "Microsoft.Extensions.DependencyInjection.Abstractions.dll",
+            "Microsoft.Extensions.DependencyInjection.dll",
+            "Microsoft.Extensions.Logging.Abstractions.dll",
+            "Microsoft.Extensions.Logging.dll",
+            "Microsoft.Extensions.Options.dll",
+            "Microsoft.Extensions.Primitives.dll",
+          )
+
+          # Define whitelist for extra DLLs (install-only)
+          $whitelistedExtraDlls = @(
+          
+          )
+
+          # Get all unique DLLs in the entire build repository
+          # Define which folders to check in the repo
+          $foldersToCheck = @(
+              "$(Get-Location)\MetaMorpheus\EngineLayer\bin\Release\net8.0",
```

Comment:
Would it be possible to not hard-code `net8.0` throughout? Could you use an environment variable like `DOTNET_VERSION`